generated/xamarin-flexible-sync/realm-todo-dotnet/realm.json

```diff
@@ -0,0 +1,4 @@
+﻿{
+  "appId": "todo-sync-jxgjv",
```

Comment:
Do we want to let the users use our server?

Reply:
Yes; it gets updated by a script before deploying to docs.